pkg/registry/registry_to_model_test.go

```diff
@@ -45,6 +47,10 @@ func testExpectedProperties() []property.Property {
 		property.MustBuildGVK("etcd.database.coreos.com", "v1beta2", "EtcdBackup"),
 		property.MustBuildGVK("etcd.database.coreos.com", "v1beta2", "EtcdRestore"),
 	}
+	props = append(props, property.Property{
+		Type:  "olm.constraint",
+		Value: json.RawMessage(constraint),
+	})
```

Comment:
Nit: rather using append in a separate statement, in-line this extra property into the original `props` initialization?

Reply:
Sounds good.